.github/CONTRIBUTING.md

```diff
@@ -765,43 +762,43 @@ For testing we use:
 
 Unit tests are located in `*.spec.ts` files such as [ `button.component.spec.ts` ](https://github.com/kirbydesign/designsystem/blob/master/libs/designsystem/src/lib/components/button/button.component.spec.ts) and integration tests in `*.spec.integration.ts` files such as [ `button.component.integration.spec.ts` ](https://github.com/kirbydesign/designsystem/blob/master/libs/designsystem/src/lib/components/button/button.component.integration.spec.ts).
 
 Remember that unit tests test a single unit - so if you find yourself relying on other components or functions without them being mocked or stubbed, then you are most likely writing an integration test. 
 
-#### The good test can be read as a sentence 
+### The good test can be read as a sentence 
 
 Being able to read a test as a sentence, makes it clear what has gone wrong when the test fails and makes it act as a functional requirement. 
 
-In the following two examples - note how the _good example_ clearly communicates intent while being easy to read and understand.
-
-Consider the following _bad_ example: 
-
-![](images/bad-test-example.png)
+Have a look at the following two examples:
 
-Here you have to interpret what is meant by: 
-
-> ButtonComponent + Kirby page Page Actions background-color: #fff 
-
-The intent of the test is unclear; what is it exactly that should have the white background color here? Not only that, but the sentence is also hard to read and understand as a sentence.
-
-Now consider the following _good_ example:
-
-![](images/good-test-example.png)
+```Javascript
+describe('ButtonComponent + Kirby Page', () => {
+  describe('Page Actions', () => {
+    it('background-color: #fff', ...) 
+  })
+})
+```
 
-When read as a sentence, the intent of the test is clearly communicated while being easy to read and understand:
+```Javascript
+describe('ButtonComponent in Kirby Page', () => {
+  describe('inside page actions', () => {
+    it('should render with correct background-color', ...) 
+  })
+})
+```
 
-> ButtonComponent in Kirby Page inside Page Actions should render with correct background-color
+Start by reading the first example as a sentence: _"ButtonComponent + Kirby Page Page Actions background-color: #fff"_. Notice how hard the test is to read and understand when read like this. The intent of the test is also unclear; what is it exactly that should have the white background color here? Page Actions? ButtonComponent? Kirby Page?
 
-There is no guessing what should have the white background color applied; the sentence clearly states that it is the ButtonComponent. 
+Now read the second example as a sentence: _"ButtonComponent in Kirby Page inside page actions should render with correct background-color"_. It can easily be read as a sentence and the intent of the test is clear. There is no guessing what should have the white background color applied; the sentence clearly states that it is the ButtonComponent. 
```

Comment:
Consider rendering the sentence as a quote as is done in the `Start describe and it blocks with a lowercase letter` section below